lib/chef/knife/vsphere_vm_snapshot.rb

```diff
@@ -67,14 +73,22 @@ def run
     end
 
     if config[:create_new_snapshot]
-      vm.CreateSnapshot_Task(name: config[:create_new_snapshot], description: '', memory: false, quiesce: false)
+	if config[:wait]
```

Comment:
I'm not sure if it's GitHub or the code, but the indentation seems to be off here. We use 2 spaces per indentation level. I think something's off here, along with the next few lines.